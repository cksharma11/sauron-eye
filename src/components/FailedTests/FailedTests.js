import React from "react";

const FailedTests = ({ detail }) => {
  const { failedTests } = detail.test;
  return (
    <ul className="failed-cases-container">
      {failedTests.map(failedTest => (
        <li key={failedTest}>{failedTest}</li>
      ))}
    </ul>
  );
};

export default FailedTests;
