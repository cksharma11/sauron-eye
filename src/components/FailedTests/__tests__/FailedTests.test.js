import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import FailedTests from "../FailedTests";

configure({ adapter: new Adapter() });

describe("FailedTests", () => {
  it("should render properly", () => {
    const wrapper = shallow(
      <FailedTests failedSuites={["Failed 1", "Failed 2"]} />
    );
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
