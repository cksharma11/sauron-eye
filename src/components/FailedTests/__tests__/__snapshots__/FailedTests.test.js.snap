// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FailedTests should render properly 1`] = `
"<ul className=\\"failed-cases-container\\">
  <li>
    Failed 1
  </li>
  <li>
    Failed 2
  </li>
</ul>"
`;
